vsintegration/src/FSharp.Editor/CodeFix/AddOpenCodeFixProvider.fs

```diff
@@ -133,6 +133,7 @@ type internal FSharpAddOpenCodeFixProvider
             
         let quilifySymbolFixes =
             candidates
+            |> Seq.filter (fun (entity,_) -> not(entity.FullRelativeName.Contains("op_"))) // Don't include fully-qualified operator names. The resultant suggestion makes the code not compile.
```

Comment:
maybe StartsWith? Is there another way we can verify this is an actual operator?